### ISSUE TITLE
new term: Respiratory distress requiring mechanical ventilation

### ISSUE THREAD

**Issue by yankeetomer** [obophenotype/human-phenotype-ontology#4321]
While there are several terms describing respiratory distress, it is hard to describe the situation of a patient requiring mechanical ventilation using HPO terms. In similarity to the term “Nasogastric tube feeding” (HP:0040288), it is sometimes necessary to describe the extreme of a certain phenotype that requires a significant external intervention. A specific situation is that of a newborn which immediately requires mechanical ventilation, a situation that is not precisely annotated using “Neonatal respiratory distress” (HP:0002643).

Therefore, I suggest a new term to be added under mode of inheritance:

### 1. Preferred term label

- Respiratory distress requiring mechanical ventilation

### 2. Synonyms

- Respiratory distress necessitating mechanical ventilation
- Respiratory distress requiring endotracheal intubation

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

**Respiratory distress requiring mechanical ventilation**
A state of respiratory distress that requires a life saving intervention in the form of gaining airway access and instituting positive pressure ventilation.

Definitions:
PMID: 18710593
PMID: 28620428

Examples of cases caused by inherited disease:
PMID: 29478781
PMID: 30215711
PMID: 29478781

### 4. Parent term
Respiratory distress
HP:0002098
(http://compbio.charite.de/hpoweb/showterm?id=HP:0002098)

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

RESPIRATORY DISTRESS SYNDROME IN PREMATURE INFANTS
OMIM: 267450

MYOPATHY, AREFLEXIA, RESPIRATORY DISTRESS, AND DYSPHAGIA, EARLY-ONSET; EMARDD
OMIM: 614399

### 6. Your nano-attribution 

Tomer Talmy M.D.
Organization: Emedgene Technologies
ORCID: https://orcid.org/0000-0002-7440-8864

**Comment by pnrobinson** [comment]
There was a term
https://hpo.jax.org/app/browse/term/HP:0004887
Respiratory failure requiring assisted ventilation

But I will add this as a synonym. Thanks for the useful info etc!

**Comment by yankeetomer** [comment]
@pnrobinson sure thing 👍
